src/main/java/me/skulduggerry/gameoflife/controller/GenerationController.java

```diff
@@ -0,0 +1,59 @@
+package me.skulduggerry.gameoflife.controller;
+
+import static me.skulduggerry.gameoflife.controller.Binding.UPLOAD_PATH;
+
+import java.io.IOException;
+
+import javax.servlet.http.HttpServletResponse;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestPart;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.multipart.MultipartFile;
+
+import lombok.extern.slf4j.Slf4j;
+import me.skulduggerry.gameoflife.exceptions.RedirectionFailedException;
+import me.skulduggerry.gameoflife.model.Generation;
+
+@RestController
+@Slf4j
+public class GenerationController {
+
+    private final ObjectMapper mapper;
+
+    @Autowired
+    public GenerationController(ObjectMapper mapper) {
+        this.mapper = mapper;
+    }
```

Comment:
Great solution! There is an even yet slimmer way of defining that. Just make sure to import Lombok's NonNull (missing in this suggestion here):
```suggestion
@RequiredArgsConstructor
public class GenerationController {

    @NonNull
    private final ObjectMapper mapper;
```